api/src/pcapi/scripts/beneficiary_fraud_check/remove_dms_content_tzinfo.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+from functools import wraps
+import logging
+import time
+
+import pytz
+from sqlalchemy import func
+from sqlalchemy.orm.attributes import flag_modified
+
+from pcapi.connectors.dms.models import parse_dms_datetime
+from pcapi.core.fraud import models as fraud_models
+from pcapi.models import db
+from pcapi.repository import repository
+
+
+logging.basicConfig()
+logger = logging.getLogger("my-logger")
+logger.setLevel(logging.DEBUG)
+
+
+def timed(func):  # type: ignore[no-untyped-def]
+    """This decorator prints the execution time for the decorated function."""
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):  # type: ignore[no-untyped-def]
+        start = time.time()
+        result = func(*args, **kwargs)
+        end = time.time()
+        logger.debug("%s ran in %ss", func.__name__, round(end - start, 2))
+        return result
+
+    return wrapper
+
+
+def clear_line(n: int = 1) -> None:
+    LINE_UP = "\033[1A"
+    LINE_CLEAR = "\x1b[2K"
+    for _ in range(n):
+        print(LINE_UP, end=LINE_CLEAR)
+
+
+def display_progress(start_time: float, current: int, total: int) -> None:
+    progression = current / total
+    elapsed_time = time.time() - start_time
+    duration = elapsed_time / progression
+    timestamp_eta = start_time + duration
+    datetime_eta = datetime.datetime.fromtimestamp(timestamp_eta).astimezone(pytz.timezone("Europe/Paris"))
+    time_eta = datetime.datetime.strftime(datetime_eta, "%H:%M:%S")
+    print(f"{current:6d} / {total} -> {progression * 100:3.2f}% in {elapsed_time:.2f}s")
+    print(f"ETA {time_eta} Estimated duration {duration:.2f}s")
+    clear_line()
+
+
+@timed
+def remove_dms_content_tzinfo(dry_run: bool = True, batch_size: int = 1000) -> None:
+    query = fraud_models.BeneficiaryFraudCheck.query.filter(
+        fraud_models.BeneficiaryFraudCheck.type == fraud_models.FraudCheckType.DMS,
+        fraud_models.BeneficiaryFraudCheck.resultContent.isnot(None),
+    ).order_by(fraud_models.BeneficiaryFraudCheck.id)
+    max_id = db.session.query(func.max(fraud_models.BeneficiaryFraudCheck.id)).scalar()
+    print(f"{max_id} beneficiary fraud checks to update")
+
+    start_index = 0
+    start_time = time.time()
+    subquery = query
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏Appelle la directement subquery :smile: